.github/workflows/simorgh-deploy-storybook.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: Install Node Modules
         if: steps.cache.outputs.cache-hit != 'true'
-        run: yarn ci
+        run: ./scripts/yarnInstallCi.sh
```

Comment:
this is my opinion but I liked explicitly seeing these steps. keen to hear what other think though

Reply:
I think the main idea behind it was to mitigate the amount changes we would have to make for each workflow if NPM were to go down, we would just need to remove the `--check-cache` parameter. But you could also argue you could do a find and replace. I'm not fussed either way, I can always raise an issue after to change this, just thinking this work is taking a long time.

Reply:
ok cool. what about naming the script `/scripts/installNodeModules.sh` so it's clearer and mean's we don't have to update the name if we ever switch to npm